src/python/pants_release/generate_github_workflows.py

```diff
@@ -879,6 +862,17 @@ def build_wheels_job(
                                     -H "Content-Type: application/octet-stream" \\
                                     ${{ needs.release_info.outputs.release-asset-upload-url }}?name=pants.$LOCAL_TAG.pex \\
                                     --data-binary "@dist/src.python.pants/pants.$LOCAL_TAG.pex"
+
+                                WHLS=$(find dist/deploy/wheels/pantsbuild.pants -type f -name "*.whl")
+                                for WHL in $WHLS; do
+                                    curl -L --fail \\
+                                        -X POST \\
+                                        -H "Authorization: Bearer ${{ github.token }}" \\
+                                        -H "Content-Type: application/octet-stream" \\
+                                        "${{ needs.release_info.outputs.release-asset-upload-url }}?name=$(basename $WHL)" \\
+                                        --data-binary "@$WHL";
+                                done
```

Comment:
Like all GitHub-Action code, this deserves an extremely careful eye. Although as we do each release, we squash bugs.

Reply:
Looks fine to me, but I can't guarantee there isn't a whitespace or quoting issue just from eyeballing.

Also, in the future it would be great if `pants publish` could do this.

Reply:
Hot damn that's an idea